### PR TITLE
proteus: added framework

### PR DESCRIPTION
**Pull request by noboomu**

### CODE REVIEW

File: frameworks/Java/proteus/setup-mysql.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+fw_depends mysql
```
Comment (martin-g) [inline frameworks/Java/proteus/setup-mysql.sh]: Is this really needed ?
setup.sh calls it anyway

File: frameworks/Java/proteus/setup.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+fw_depends postgresql mysql java maven
+
+sed -i 's|mysql://.*:3306|mysql://'"${DBHOST}"':3306|g' conf/application.conf
+sed -i 's|postgresql://.*:3306|postgresql://'"${DBHOST}"':3306|g' conf/application.conf
```
Comment (martin-g) [inline frameworks/Java/proteus/setup.sh]: Is it 3306 or 5432 ?

Comment (noboomu) [inline frameworks/Java/proteus/setup.sh]: 5432. sigh... what's the easiest way on your end for a fix?

File: frameworks/Java/proteus/setup-mysql.sh (hunk 1 above)
Comment (noboomu) [inline frameworks/Java/proteus/setup-mysql.sh]: Nope. Just going off the work of others.

**Comment by noboomu** [comment]
Okay @martin-g, cleaned that up. Thanks!

**Pull request closed without merging**